core/resources/scene/config.yaml

```diff
@@ -177,7 +183,10 @@ layers:
         data: {source: osm, layer: roads}
         filter: { name: true, aeroway: false, tunnel: false, railway: false, not: { kind: rail }, $zoom: { min: 10 } }
         draw:
-            text: {}
+            text: 
+                font:
+                    typeface: Regular 15 Roboto
```

Comment:
1. Based on the documentation [here](https://github.com/tangrams/tangram-docs/blob/ec9491758ad578b5607e7549336fdac07a64475b/pages/draw.md#font-parameters), there are 2 ways of specifying a font size, one is the "size" property and other is the size in the "typeface" property.

Also,
2. Instead of having typeface as a string of different values, will it make sense to have it broken down as yaml key value pairs?
```
text:
      font:
            name:
            face:
            size:
            weight:
```

This could be helpful in style mixing, where a sublayer only overrides one of the specific font properties, instead of specifying the whole typeface string.